### PR TITLE
Code Cleanup - RuntimeException

### PR DESCRIPTION
**Pull request by pirouet**
Ensure files that use RuntimeException have the appropriate use
statement included

### CODE REVIEW

**Comment by pirouet** [comment]
Travis showing error (couldn't connect to GitHub). This can be seen passing under https://travis-ci.org/pirouet/my-cakephp/builds/56959662

**Comment by dereuromark** [comment]
:+1:

File: src/ORM/Association/BelongsToMany.php [hunk 1]
```diff
@@ -301,7 +302,7 @@ protected function _joinCondition($options)
      * @param \Cake\ORM\Query $fetchQuery The query to get results from
      * @param array $options The options passed to the eager loader
      * @return array
-     * @throws \RuntimeException when the association property is not part of the results set.
+     * @throws RuntimeException when the association property is not part of the results set.
```
Comment (markstory) [inline src/ORM/Association/BelongsToMany.php]: I think doc blocks need absolute class references to generate API docs properly.

Comment (ADmad) [inline src/ORM/Association/BelongsToMany.php]: Yes doc blocks should have absolute class references. The API generator does not have a problem but some IDEs take it as relative path.

Comment (pirouet) [inline src/ORM/Association/BelongsToMany.php]: Absolute path references updated

**Comment by markstory** [comment]
Thanks :+1:

**Pull request merged**